From f37fa162020846da5950eba704e58e41889b9d3a Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Thu, 17 May 2012 10:59:18 +0200
Subject: [PATCH 43/79] cpsw,phy_device: demote PHY message to INFO

This keeps them from appearing on the screen when booting with 'quiet'

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
---
 drivers/net/ethernet/ti/cpsw.c |    2 +-
 drivers/net/phy/phy_device.c   |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/ti/cpsw.c b/drivers/net/ethernet/ti/cpsw.c
index 86fe57d..d10996d 100644
--- a/drivers/net/ethernet/ti/cpsw.c
+++ b/drivers/net/ethernet/ti/cpsw.c
@@ -638,7 +638,7 @@ static void cpsw_slave_open(struct cpsw_slave *slave, struct cpsw_priv *priv)
 		    slave->data->phy_id, slave->slave_num);
 		slave->phy = NULL;
 	} else {
-		printk(KERN_ERR"\nCPSW phy found : id is : 0x%x\n",
+		printk(KERN_INFO"\nCPSW phy found : id is : 0x%x\n",
 			slave->phy->phy_id);
 		cpsw_set_phy_config(priv, slave->phy);
 		phy_start(slave->phy);
diff --git a/drivers/net/phy/phy_device.c b/drivers/net/phy/phy_device.c
index 83a5a5a..8b84789 100644
--- a/drivers/net/phy/phy_device.c
+++ b/drivers/net/phy/phy_device.c
@@ -380,7 +380,7 @@ struct phy_device * phy_connect(struct net_device *dev, const char *bus_id,
 	 * PHY with the requested name */
 	d = bus_find_device_by_name(&mdio_bus_type, NULL, bus_id);
 	if (!d) {
-		pr_err("PHY %s not found\n", bus_id);
+		pr_info("PHY %s not found\n", bus_id);
 		return ERR_PTR(-ENODEV);
 	}
 	phydev = to_phy_device(d);
@@ -503,7 +503,7 @@ struct phy_device *phy_attach(struct net_device *dev,
 	 * PHY with the requested name */
 	d = bus_find_device_by_name(bus, NULL, bus_id);
 	if (!d) {
-		pr_err("PHY %s not found\n", bus_id);
+		pr_info("PHY %s not found\n", bus_id);
 		return ERR_PTR(-ENODEV);
 	}
 	phydev = to_phy_device(d);
-- 
1.7.10

